python/ray/data/_internal/stats.py

```diff
@@ -631,6 +631,9 @@ def __init__(
         self.global_bytes_restored: int = 0
         self.dataset_bytes_spilled: int = 0
 
+        # Streaming split iterator stats
```

Comment:
Let's also note in the comment that this is measured at Dataset level (as opposed to operator level)
```suggestion
        # Streaming split coordinator stats
```